### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by seanmrnda** [adityatelange/hugo-PaperMod#913] [General]
The documentation is not clear on how to use the Icons.

**Comment by adityatelange** [comment]
Don't use

**Reply by seanmrnda** [reply]
what? why would you say that?

**Reply by seanmrnda** [reply]
Look, I find your work interesting and I want to try it. I just can't figure out why I can't use params keywords on config.yml

**Reply by adityatelange** [reply]
Alright then pls head over to exampleSite/config.yml before asking.

**Reply by adityatelange** [reply]
> what? why would you say that?

Thing is almost everything is present somewhere. People just don't take efforts to do that, coz asking someone and getting the solution is easier and quicker.
Also there's no one trying to help others when they can. This makes maintainer feel on the other side always whenever someone asks a silly doubt.

**Reply by seanmrnda** [reply]
I'm really sorry. T_T

**Comment by seanmrnda** [comment]
![config-yml](https://user-images.githubusercontent.com/45195351/168202957-b957a8cd-0254-46f6-9caf-74d5de189118.png)

I can't seem to make this config work. I need help.

**Reply by adityatelange** [reply]
https://github.com/adityatelange/hugo-PaperMod/blob/241f6697e01a966d6e8e503eee322fc6029a4880/config.yml#L141-L158

**Reply by seanmrnda** [reply]
Thanks, idk what happened but I simply copied that format and it works now.

**Reply by adityatelange** [reply]
> Thanks, idk what happened but I simply copied that format and it works now.

Spaces sir!
